lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_packages/recipes/default.rb

```diff
@@ -17,6 +17,16 @@
 # limitations under the License.
 #
 
+# Install common packages
+%w(install upgrade remove).each do |act|
```

Comment:
minor suggestion here and for line 31: do ``remove`` first.  I could imagine some scenario where you want to remove some conflicting package to replace it with another.